app/src/main/java/org/supla/android/MainActivity.java

```diff
@@ -126,79 +123,96 @@ private boolean SetListCursorAdapter() {
 
         if (channelListViewCursorAdapter == null) {
 
-            channelListViewCursorAdapter = new ListViewCursorAdapter(this, DbH_ListView.getChannelListCursor());
+            channelListViewCursorAdapter = new ListViewCursorAdapter(this, getDbHelper().getChannelListCursor());
```

Comment:
The channel list cannot display those that have no assigned function.
https://github.com/SUPLA/supla-android/blob/master/app/src/main/java/org/supla/android/db/DbHelper.java#L1548

Reply:
Should be fixed.